spec/author_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rspec'
```

Comment:
Please check the unit tests as they are failing when I run `rspec`

![image](https://user-images.githubusercontent.com/41517876/219127900-31df51ea-eaa4-4321-88c9-de2bdfb874a1.png)

Reply:
hi @momotsow, thank you for notice that, i will fix it.